scripts/import/import_clinvar_xml

```diff
@@ -672,33 +690,52 @@ sub import{
         @{$record->{feature_info}}{qw/hgvs_g Chr start end gene/} = @{$record->{feature_info}->{$rsID}}{qw/hgvs_g Chr start end gene/};
       }
 
-      ($feature_object,  $feat, $alt_allele) = get_variant($record, $rs);
+      ($feature_object, $feat, $alt_allele) = get_variant($record, $rs);
       next unless defined $feature_object;
 
-      ## add phenotype evidence attrib to variation (either existing one or a newly inserted one)
-      $feature_object->add_evidence_value($phenotype_evidence);
-      $variation_adaptor->update($feature_object);
-
-      ## add synonym
-      add_synonyms($feature_object, $record->{clinvar_variant_id}, $source) if defined $feature_object;
-      add_synonyms($feature_object, $record->{Acc}, $source) if defined $feature_object;
-      add_synonyms($feature_object, $record->{feature_info}{$omimstr}, $omim_source) if defined $feature_object && defined $record->{feature_info}{$omimstr};
+      add_clinvar_data($feature_object, $feat, $alt_allele, $record);
 
-      ## add phenotype_feature & attrib (there may not be an alt_allele)
-      import_phenotype_feature($record, $feature_object, 'Variation', $feat, $alt_allele);
     }
   }
-  elsif(defined  $record->{feature_info}->{dbVar}){
+  elsif(!defined $structvar && $try_match ){
+    ($feature_object, $feat, $alt_allele) = get_variant($record, undef);
+    next unless defined $feature_object;
+
+    add_clinvar_data($feature_object, $feat, $alt_allele, $record);
+  }
```

Comment:
When I test with a variant that isn't in the database, I get these messages:

Exiting subroutine via next at scripts/import/import_clinvar_xml line 702.
Exiting eval via next at scripts/import/import_clinvar_xml line 702.
Exiting subroutine via next at scripts/import/import_clinvar_xml line 702.

They aren't very useful and could make it hard to spot other warnings. It's more useful to list the variants which could not be found.